File: java/kode program/Wilayah.java
import java.util.ArrayList;

// Wilayah berisi beberapa Hutan
public class Wilayah {
    private String nama;
    private ArrayList<Hutan> hutanList = new ArrayList<>();

    public Wilayah(String nama) {
        this.nama = nama;
    }

    public void tambahHutan(Hutan h) {
        hutanList.add(h);
    }

    public void tampilkan() {
        System.out.println("=== Wilayah: " + nama + " ===");
        for (Hutan h : hutanList) {
            h.tampilkan();
        }
    }
}

File: python/kode program/wilayah.py
# ============================================
# Kelas Wilayah menyimpan beberapa hutan
# ============================================
# Kelas Wilayah berisi kumpulan Hutan
class Wilayah:
    def __init__(self, nama):
        self.nama = nama
        self.hutan_list = []

    def tambah_hutan(self, hutan):
        self.hutan_list.append(hutan)

    def tampilkan(self):
        print(f"\n=== Wilayah: {self.nama} ===")
        for h in self.hutan_list:
            h.tampilkan()

